### PR TITLE
chore(other): update node to 18.x multiarch 

### PR DESCRIPTION
**Pull request by daragh-king-genesys**

### CODE REVIEW

File: packages/genesys-spark-components/src/components/beta/gux-time/tests/__snapshots__/gux-time.spec.ts.snap [hunk 1]
```diff
@@ -6875,7 +6875,7 @@ exports[`gux-time #render using regional dates #render different time zones shou
 exports[`gux-time #render using regional dates #render different time zones should work as expected for "Antarctica/Casey" 1`] = `
 <gux-time-beta datetime="2022-07-07T12:00:00.000Z" format="full" time-zone="Antarctica/Casey">
   <mock:shadow-root>
-    11:00:00 PM Casey Time
+    8:00:00 PM Australian Western Standard Time
```
Comment (daragh-king-genesys) [inline packages/genesys-spark-components/src/components/beta/gux-time/tests/__snapshots__/gux-time.spec.ts.snap:6878]: https://timezonedb.com/time-zones/Antarctica/Casey
After Thursday, 09 March, 2023 02:59:59 AM: Clocks were moved backward to become Thursday, 09 March, 2023 12:00:00 AM

File: packages/genesys-spark-components/src/components/beta/gux-time/tests/__snapshots__/gux-time.spec.ts.snap [hunk 2]
```diff
@@ -2467,7 +2467,7 @@ exports[`gux-time #render using non regional dates #render different time zones
 exports[`gux-time #render using non regional dates #render different time zones should work as expected for "Antarctica/Vostok" 1`] = `
 <gux-time-beta datetime="2022-07-07T12:00:00.000Z" format="full" time-zone="Antarctica/Vostok">
   <mock:shadow-root>
-    6:00:00 PM Vostok Time
+    7:00:00 PM Vostok Time
```
Comment (daragh-king-genesys) [inline packages/genesys-spark-components/src/components/beta/gux-time/tests/__snapshots__/gux-time.spec.ts.snap:2470]: https://timezonedb.com/time-zones/Antarctica/Vostok
After Monday, 18 December, 2023 01:59:59 AM: Clocks were moved backward to become Monday, 18 December, 2023 12:00:00 AM

File: web-apps/genesys-spark-examples/Jenkinsfile [hunk 3]
```diff
@@ -13,7 +13,7 @@ webappPipeline {
     team = 'Core UI'
     mailer = 'matthew.cheely@genesys.com, daragh.king@genesys.com, jordan.stith@genesys.com, thomas.dillon@genesys.com, katie.bobbe@genesys.com, gavin.everett@genesys.com, jason.evans@genesys.com'
     chatGroupId = 'adhoc-30ab1aa8-d42e-4590-b2a4-c9f7cef6d51c'
-    nodeVersion = '18.16.1'
+    nodeVersion = '18.x multiarch'
```
Comment (321gillian) [inline web-apps/genesys-spark-examples/Jenkinsfile:16]: What's the multiarch bit for?

Comment (daragh-king-genesys) [reply web-apps/genesys-spark-examples/Jenkinsfile:16]: More of a devops concern. It allows them to have one version of node that they can install on different machine types / architectures.

File: packages/genesys-spark-components/src/components/beta/gux-date-time/tests/__snapshots__/gux-date-time.spec.ts.snap [hunk 4]
```diff
@@ -19,7 +19,7 @@ exports[`gux-date-time #render using non regional dates #render different dateti
 exports[`gux-date-time #render using non regional dates #render different datetime formats eg(short,medium,full,long) should render component as expected (1) 3`] = `
 <gux-date-time-beta datetime="2022-07-07T13:35:30.100Z" format="short">
   <mock:shadow-root>
-    07.07.2022 13.35
+    07.07.2022, 13.35
```
[the remark this answers is not in the document]
Comment (jordanstith15) [reply packages/genesys-spark-components/src/components/beta/gux-date-time/tests/__snapshots__/gux-date-time.spec.ts.snap:22]: I could be wrong.. but I think node actually determines a lot of how date times work when you use it

[the remark this answers is not in the document]
Comment (321gillian) [reply packages/genesys-spark-components/src/components/beta/gux-date-time/tests/__snapshots__/gux-date-time.spec.ts.snap:22]: Thanks Jordan, that was really helpful!

**Review by 321gillian — APPROVED** [review]

**Pull request merged**